gateway/requests/requests.go

```diff
@@ -29,6 +29,32 @@ type CreateFunctionRequest struct {
 	Constraints []string `json:"constraints"`
 }
 
+// UpdateFunctionRequest updates a function in the swarm.
+type UpdateFunctionRequest struct {
```

Comment:
Question - if this is the same as CreateFunctionRequest, do we need a separate physical type to unmarshal into?

Reply:
I'd put it in with the assumption that we'd end up wanting to pass more params in, will stick with the `CreateFunctionRequest` until we actually need it.

Reply:
Thank you.